celery/app/task.py

```diff
@@ -885,8 +885,13 @@ def replace(self, sig):
             )
 
         if self.request.chain:
+            options = sig.options.copy()
             for t in reversed(self.request.chain):
                 sig |= signature(t, app=self.app)
+            sig.set(
+                link=options['link'],
+                link_error=options['link_error']
```

Comment:
```suggestion
                link=options.get('link', None),
                link_error=options.get('link_error', None)
```

It turns out both `link` and `link_error` may not be present.